src/CargoBuild/sdk/Sdk.props

```diff
@@ -0,0 +1,94 @@
+		<ManagedLanguageTargetsGotImported Condition="'$(MSBuildProjectExtension)' == '.vbproj'">true</ManagedLanguageTargetsGotImported>
+		<ManagedLanguageTargetsGotImported Condition="'$(MSBuildProjectExtension)' == '.fsproj'">true</ManagedLanguageTargetsGotImported>
+	</PropertyGroup>
+
+	<Import Project="$(CustomBeforeNoTargetsProps)" Condition=" '$(CustomBeforeNoTargetsProps)' != '' And Exists('$(CustomBeforeNoTargetsProps)') " />
+
+	<PropertyGroup>
+		<!-- Disable default Compile and EmbeddedResource items for NoTargets projects -->
+		<EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+		<EnableDefaultEmbeddedResourceItems>false</EnableDefaultEmbeddedResourceItems>
+
+		<!--
+      NuGet should always restore Traversal projects with "PackageReference" style restore.  Setting this property will cause the right thing to happen even if there aren't any PackageReference items in the project.
+    -->
+		<RestoreProjectStyle>PackageReference</RestoreProjectStyle>
+
+		<!-- Targeting packs shouldn't be referenced as traversal projects don't compile. -->
+		<DisableImplicitFrameworkReferences Condition="'$(DisableImplicitFrameworkReferences)' == ''">true</DisableImplicitFrameworkReferences>
+
+		<!-- Disable publish actions -->
+		<CopyBuildOutputToPublishDirectory Condition="'$(CopyBuildOutputToPublishDirectory)' == ''">false</CopyBuildOutputToPublishDirectory>
+		<CopyOutputSymbolsToPublishDirectory Condition="'$(CopyOutputSymbolsToPublishDirectory)' == ''">false</CopyOutputSymbolsToPublishDirectory>
+
+		<!-- Don't generate a deps file -->
+		<GenerateDependencyFile Condition="'$(GenerateDependencyFile)' == ''">false</GenerateDependencyFile>
+
+		<!-- Don't generate assembly info -->
+		<GenerateAssemblyInfo Condition="'$(GenerateAssemblyInfo)' == ''">false</GenerateAssemblyInfo>
+
+		<!-- Don't generate editor config file -->
+		<GenerateMSBuildEditorConfigFile Condition="'$(GenerateMSBuildEditorConfigFile)' == ''">false</GenerateMSBuildEditorConfigFile>
+
+		<!-- Don't log the high priority message mentioning this project's name (or copy the product we didn't build). -->
+		<SkipCopyBuildProduct Condition="'$(SkipCopyBuildProduct)' == ''">true</SkipCopyBuildProduct>
+
+		<!-- Don't automatically reference assembly packages since NoTargets don't need reference assemblies -->
+		<AutomaticallyUseReferenceAssemblyPackages Condition="'$(AutomaticallyUseReferenceAssemblyPackages)' == ''">false</AutomaticallyUseReferenceAssemblyPackages>
+		<NoCompilerStandardLib Condition="'$(NoCompilerStandardLib)' == ''">false</NoCompilerStandardLib>
+		<NoStdLib Condition="'$(NoStdLib)' == ''">true</NoStdLib>
+
+		<!-- Disable Visual Studio's Fast Up-to-date Check and rely on MSBuild to determine -->
+		<DisableFastUpToDateCheck Condition="'$(DisableFastUpToDateCheck)' == ''">true</DisableFastUpToDateCheck>
+	</PropertyGroup>
+
+	<ItemDefinitionGroup Condition=" '$(NoTargetsDoNotReferenceOutputAssemblies)' != 'false' ">
```

Comment:
Bad copy/pasted property name.

Reply:
I'm actually not convinced that we need the condition at all? Since you never would want to reference the output assembly in the normal way.